src/validation.cpp

```diff
@@ -3523,7 +3523,7 @@ bool static LoadBlockIndexDB(const CChainParams& chainparams)
     // Check whether we need to continue reindexing
     bool fReindexing = false;
     pblocktree->ReadReindexing(fReindexing);
-    fReindex |= fReindexing;
+    fReindex = fReindex | fReindexing;
```

Comment:
Might as well use the atomic or here since we're making it atomic, no?

Reply:
`|=`? It is only defined on atomic integral types I believe, not bool

Reply:
Err, indeed, quivalent is to do an if(fReindexing) fReindex = true

Reply:
Yeah true that's cleaner, will do